doc/Makefile

```diff
@@ -36,6 +37,11 @@ clean:
 	-rm -rf _build auto_examples auto_tutorials generated *.stc *.fif *.nii.gz
 
 html_stable:
+	$(MPROF) -b html $(ALLSPHINXOPTS) _build/html_stable
+	@echo
+	@echo "Build finished. The HTML pages are in _build/html_stable."
+
+html_stable-memory:
```

Comment:
Naive question: `html_stable-memory` just uses `$(SPHINXBUILD)` and `html_stable` uses `$(MPROF)`. Is it intended? I would assume only the new `-memory` targets uses `mprof`.

Reply:
Whoops, that's backward!